system/shaders/GLES/3.0/gles_yuv2rgb_basic.frag

```diff
@@ -0,0 +1,79 @@
+#version 300 es
+
+precision mediump float;
+
+uniform sampler2D m_sampY;
+uniform sampler2D m_sampU;
+uniform sampler2D m_sampV;
+in vec2 m_cordY;
+in vec2 m_cordU;
+in vec2 m_cordV;
+uniform vec2 m_step;
+uniform mat4 m_yuvmat;
+uniform float m_stretch;
+uniform float m_alpha;
+out vec4 fragColor;
+
+void main()
+{
+  vec4 rgb;
+#if defined(XBMC_YV12) || defined(XBMC_NV12)
+
+  vec4 yuv;
+  yuv.rgba = vec4( texture(m_sampY, m_cordY).r
```

Comment:
GLES 3.1 would be better here, saves a lot of work for texture lookup: https://www.khronos.org/registry/OpenGL-Refpages/es3.1/html/textureGather.xhtml

Reply:
Does it hurt to require 3.1 instead of 3.0?